src/components/resourceLinks/ResourceLinks.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import "./ResourceLinks.css"
+const ResourceLinks = ({icon, icon_alt, target, url, icon_only, name, classname}) => {
+    return (
+        <div>
+            <a className="resource-link" href={url} target={target}>
+            {icon_only && icon &&
+                <img className={classname} src={icon} alt={icon_alt}/>
+            }
+            {!icon_only &&
+                name}
```

Comment:
`name` is misschien iets te vaag. Het is immers de link-tekst of de link-label, dus een beschrijvende property zou hier beter zijn!